test/regression/RegressionTest.cc

```diff
@@ -38,38 +38,24 @@ void RegressionTest::run(const char* source, const char* triple,
   memory_ = std::make_shared<simeng::memory::SimpleMem>(memorySize);
 
   // Initialise a SimOS object.
-  simeng::OS::SimOS OS = simeng::OS::SimOS(DEFAULT_STR, {}, memory_, true);
+  simeng::OS::SimOS OS = simeng::OS::SimOS(
+      memory_, simeng::span<char>(reinterpret_cast<char*>(code_), codeSize_));
 
   // Create a Process from the assembled code block.
   // Memory allocation for process images also takes place
-  // during Process creation. The Elf binary is parsed
-  // and relevant sections are copied to the process image.
+  // during Process creation.
   // The process image is finalised by the createStack method
   // which creates and populates the initial process stack.
```

Comment:
I feel like this comment is misplaced now. Doesn't really pertain to the code beneath it.